shared/lib/url-creator.js

```diff
@@ -1,18 +1,24 @@
 'use strict';
 
 const urlJoin = require('url-join');
+const { URLSearchParams } = require('url');
 
 class URLCreator {
   constructor(basePath) {
     this._basePath = basePath;
   }
 
-  archiveUrlForDate(roomId, date) {
+  archiveUrlForDate(roomId, date, { viaServers = [] } = {}) {
+    let qs = new URLSearchParams();
```

Comment:
```
URLSearchParams is not a constructor
```

Reply:
This got weirder. It doesn't work on macOS with Chrome Version 103.0.5060.114 but it does work on Windows with the same Chrome version 🤔

Reply:
Works well without importing and using the global `URLSearchParams` ✅